SemanticMediaWiki.php

```diff
@@ -52,7 +52,9 @@
 
 if ( $wgDBname === 'constantnoblewiki' ) {
 	array_push( $smwgPageSpecialProperties, '_CDAT' );
-
+	
+	$smwgDVFeatures = $smwgDVFeatures | SMW_DV_PVUC;
+	
```

Comment:
```suggestion

	$smwgDVFeatures = $smwgDVFeatures | SMW_DV_PVUC;

```
(I swear I'm not picky)